README.md

```diff
@@ -1,6 +1,7 @@
 # MongooseIM platform
 
-[![Build Status](https://travis-ci.org/esl/MongooseIM.svg?branch=master)](https://travis-ci.org/esl/MongooseIM)
+[![Build Status](https://github.com/esl/MongooseIM/workflows/CI/badge.svg)](https://github.com/esl/MongooseIM/actions)
+[![CircleCI](https://circleci.com/gh/esl/MongooseIM.svg?style=shield&circle-token=53cdda11cfb95bd7a7044c509c89cb73d3fdeb5a)](https://github.com/esl/MongooseIM/)
```

Comment:
Shouldn't it link to our circleci page? The same applies to `doc/index.md`.

Reply:
The circleci page is private, we're paying for a private account and only we have access. I've created a token within CircleCI (`token=53cdda11cfb95bd7a7044c509c89cb73d3fdeb5a` in the url) that has permissions only to see the status.